### PR TITLE
Hot fix add trace

### PR DESCRIPTION
**Pull request by VeraZab**
fixes: https://github.com/plotly/streambed/issues/11752

### CODE REVIEW

**Comment by VeraZab** [comment]
@dmt0 please review

**Comment by nicolaskruchten** [comment]
:dancer:

**Comment by nicolaskruchten** [comment]
How does prevtrace not have a type?? Or not exist?

**Comment by nicolaskruchten** [comment]
I guess if pushed via api and relying on default type=scatter

File: src/lib/customTraceType.js [hunk 1]
```diff
@@ -97,6 +97,6 @@ export function traceTypeToPlotlyInitFigure(traceType, gl = '') {
         type: 'cone',
       };
     default:
-      return {type: traceType};
+      return {type: traceType ? traceType : 'scatter'};
```
Comment (nicolaskruchten) [inline src/lib/customTraceType.js]: This is actually problematic

Comment (nicolaskruchten) [reply src/lib/customTraceType.js]: The scatter case should set the same options as at the top otherwise there are two “scatter” paths

**Comment by nicolaskruchten** [comment]
Dancer only when scatter path is centralized :)

**Comment by nicolaskruchten** [comment]
💃 looks good to me now!

**Pull request merged**